app/lib/katello/resources/candlepin.rb

```diff
@@ -8,42 +8,6 @@ module Resources
     module Candlepin
       TOTAL_COUNT_HEADER = :x_total_count # as parsed by rest_client
 
-      class Proxy
```

Comment:
I think you can remove the `require 'katello/util/data'` but keep the `http_proxy`.